lib/babe/errors.go

```diff
@@ -71,6 +71,7 @@ var (
 	errInvalidSlotTechnique       = errors.New("invalid slot claiming technique")
 	errNoBABEAuthorityKeyProvided = errors.New("cannot create BABE service as authority; no keypair provided")
 	errLastDigestItemNotSeal      = errors.New("last digest item is not seal")
+	errLaggingSlot                = errors.New("cannot claim slot, current slot is smaller than slot of best block")
```

Comment:
nit the `claim slot` part seems out of scope given the error var name?

Perhaps just have it as 

```suggestion
	errLaggingSlot                = errors.New("current slot is smaller than slot of best block")
```

or rename the error variable name